plugins/woocommerce/includes/wc-ipp-functions.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
As earlier, we generally don't want to add new code to `includes/`. We can make exceptions of course: but do you think there might be a case here for setting this up as a set of static methods belonging to a new class in our `Automattic\WooCommerce` namespace/the `Internal` sub-namespace)?

Since that would also give us the advantage of autoloading, we could ditch [this line](https://github.com/woocommerce/woocommerce/pull/34467/files#diff-0095ea46d160c94a1276a0eed1d86a5dbc91387c39ac46aae6abd0a1d1d314b9R35).